config.py

```diff
@@ -46,10 +48,10 @@ def main():
         repos_data['repos'] = [dir_path]
         yaml.safe_dump(repos_data, open('./sysconfigs/repos.yaml', 'w'), default_flow_style=False)
         print('Installing repos.yaml on ' + args.reposdir)
-        os.popen('cp ' + dir_path + '/sysconfigs/repos.yaml ' + args.reposdir)
+        shutil.copy(dir_path + '/sysconfigs/repos.yaml', args.reposdir)
```

Comment:
Nice!
For the future we could consider migrating to Python's pathlib: https://docs.python.org/3/library/pathlib.html

Apparently, it allows one to work with paths very nicely: https://realpython.com/python-pathlib/

But this is probably overkill for this PR.